Sources/UIViewExtensions.swift

```diff
@@ -565,9 +565,34 @@ extension UIView {
         self.layer.mask = mask
     }
 
-    /// EZSwiftExtensions
+    /**
+     EZSwiftExtensions - Mask square/rectangle UIView with a circular/capsule cover
+     */
     public func roundView() {
-        self.layer.cornerRadius = min(self.frame.size.height, self.frame.size.width) / 2
+        self.setCornerRadius(radius: min(self.frame.size.height, self.frame.size.width) / 2)
+        self.layer.borderWidth = 0
+    }
+    
+    /**
+     EZSwiftExtensions - Mask square/rectangle UIView with a circular/capsule cover, with a border of desired color and width around it
+     
+     - Parameter withBorderColor: Color of circuler border
+     - Parameter withBorderWidth: Width of circuler border
+     */
+    public func roundView(withBorderColor color: UIColor, withBorderWidth width: CGFloat) {
+        self.setCornerRadius(radius: min(self.frame.size.height, self.frame.size.width) / 2)
+        self.layer.masksToBounds = true
+        self.layer.borderWidth = width
+        self.layer.borderColor = color.cgColor
+    }
+    
+    /**
+     EZSwiftExtensions - Remove all masking around UIView
+     */
+    public func nakedView() {
```

Comment:
Ah I used it for a rounded UIImageView showing a user's profile picture.

When user logs in and his/her profile picture is loaded, the mask needs to be on to make the image rounded, and with a border around it.
When user logs out, the image will change to a image asset, and the mask needs to be taken out.

I thought a helper method to remove any mask present would be helpful.